README.md

```diff
@@ -1,3 +1,5 @@
 # link-preview-expander (Sourcegraph extension)
 
-.
+Show hover previews for links
```

Comment:
We should have a bit more content here. Ideas:
- Explain that it uses OpenGraph
- Add a screenshot specifically showing what it looks like for a GitHub issues

Reply:
@tjkandala this was not resolved